content/run.md

```diff
@@ -240,3 +240,111 @@ Now you can perform the final steps to build and submit the case to the job queu
 
     [~/cases/SCA_SpRun]$ ./case.build
     [~/cases/SCA_SpRun]$ ./case.submit
+
+### Example regional run bash scripts
+
+`````{tabs}
+  ````{tab} 0.25&deg; grid
+
+  The following example requires read access to
+  `/cluster/projects/nn9373k/data_regional_SBS` on Fram.
```

Comment:
I think it would be good if this script is consistent with the regional steps explained [here](https://metos-uio.github.io/CTSM-Norway-Documentation/run/#run-a-regional-case) for the same resolution. Otherwise, this will bring confusion to the users (especially for beginners). The domain and surface data files for the Scandinavia region at 0.5 resolution are available in the common shared folder (/cluster/shared/noresm/inputdata/share/domains and /cluster/shared/noresm/inputdata/lnd/clm2/surfdata_map). You may use these files in the script.

Reply:
There is no `subset_data.py` script in the [NordicESMhub version of CTSM](https://github.com/NordicESMhub/ctsm/tree/master/tools/contrib) used in this script. So to reproduce the 0.5 Scandinavia simulations, the subset domain and surface data files are needed. Are they `/cluster/shared/noresm/inputdata/share/domains/domain.0.5x0.5_scand.nc` and `/cluster/shared/noresm/inputdata/lnd/clm2/surfdata_map/surfdata_0.5_scand_hist_16pfts_Irrig_CMIP6_simyr2000_c190814.nc`?

Reply:
Should the new script be in addition to the existing one? Or should the existing one be removed and replaced by the new one?

Reply:
Just one script is enough because all steps are the same and just need to replace the domain and surface files for a different resolution run. Maybe at the end, you may mention, that if the user wishes to run on 0.25 resolution they can use those 0.25-degree domains and surface data files. But provide the location of those files to access for any user not specific to project users.

Reply:
I have tried with
- `/cluster/shared/noresm/inputdata/share/domains/domain.0.5x0.5_scand.nc` (domain file)
- `/cluster/shared/noresm/inputdata/lnd/clm2/surfdata_map/surfdata_0.5_scand_hist_16pfts_Irrig_CMIP6_simyr2000_c190814.nc` (surface data file)

but running the case terminates with an error apparently related to surface data reading. Attached are the updated script and the log file:
- [cesm.log.2932786.210623-180608](https://github.com/MetOs-UiO/CTSM-Norway-Documentation/files/6703632/cesm.log.2932786.210623-180608.txt)
- [steps_reg_run_emerald-CLM_0.5_Scand.sh](https://github.com/MetOs-UiO/CTSM-Norway-Documentation/files/6703633/steps_reg_run_emerald-CLM_0.5_Scand.sh.txt)

I guess this is caused by not [subsetting the regional data](https://metos-uio.github.io/CTSM-Norway-Documentation/run/#domain-and-surface-data). However, the script `$CTSM_ROOT/tools/contrib/subset_data.py` is not there in the cloned CTSM version (`git clone -b release-clm5.0 https://github.com/NordicESMhub/ctsm.git`); and I am not sure about how to use [`subset_data.py` linked in the documentation](https://github.com/ESCOMP/CTSM/blob/master/tools/contrib/subset_data.py) because it contains several [hard-coded paths](https://github.com/ESCOMP/CTSM/blob/master/tools/contrib/subset_data.py#L972-L1120), which would be tricky to be modified in a bash script.

Reply:
How about keeping the script as it is, but add the comments from [here](https://github.com/MetOs-UiO/CTSM-Norway-Documentation/issues/24#issuecomment-861592724) above to draw attention to the differences?

Reply:
Yes: I have [added those comments before the script](https://github.com/MetOs-UiO/CTSM-Norway-Documentation/commit/f84345dc1419812772d3512e36609d6b8faf4109#diff-2d4a04d5d29284198da640dc1e2c759ca167cabacfa03d174ac49f7b05a853a5R247-R257)

Reply:
> I have tried with
> 
> * `/cluster/shared/noresm/inputdata/share/domains/domain.0.5x0.5_scand.nc` (domain file)
> * `/cluster/shared/noresm/inputdata/lnd/clm2/surfdata_map/surfdata_0.5_scand_hist_16pfts_Irrig_CMIP6_simyr2000_c190814.nc` (surface data file)
> 
> but running the case terminates with an error apparently related to surface data reading. Attached are the updated script and the log file:
> 
> * [cesm.log.2932786.210623-180608](https://github.com/MetOs-UiO/CTSM-Norway-Documentation/files/6703632/cesm.log.2932786.210623-180608.txt)
> * [steps_reg_run_emerald-CLM_0.5_Scand.sh](https://github.com/MetOs-UiO/CTSM-Norway-Documentation/files/6703633/steps_reg_run_emerald-CLM_0.5_Scand.sh.txt)
> 
> Perhaps this is caused by not [subsetting the regional data](https://metos-uio.github.io/CTSM-Norway-Documentation/run/#domain-and-surface-data). However, the script `$CTSM_ROOT/tools/contrib/subset_data.py` is not there in the cloned CTSM version (`git clone -b release-clm5.0 https://github.com/NordicESMhub/ctsm.git`); and I am not sure about how to use [`subset_data.py` linked in the documentation](https://github.com/ESCOMP/CTSM/blob/master/tools/contrib/subset_data.py) because it contains several [hard-coded paths](https://github.com/ESCOMP/CTSM/blob/master/tools/contrib/subset_data.py#L972-L1120), which would be tricky to be modified in a bash script.

Try running again with the same domain and surface data files. I have replaced with the files I ran successfully CTSM. The NordicESMhub does not have the latest version from ESCOMP and hence `subset_data.py` file is missing there. I think this script is recently updated at NCAR and this is very useful to create input data sets for regional simulations when compared to complex _ncl_ scripts.

Reply:
Thanks for updating the files: the model ran successfully. The bash script for Scandinavia at 0.5 degree resolution is added as first tab (ab64a9dd1a85bf5aa9cad9664c612c221c5a1636)

Reply:
LGTM!